services/minespace-web/src/components/Forms/projects/majorMineApplication/MajorMineApplicationForm.tsx

```diff
@@ -0,0 +1,382 @@
+        }}
+        onSubmit={handleSubmit}
+      >
+        <Row>
+          <Col span={24}>
+            <Typography.Title level={4}>Basic Information</Typography.Title>
+            <Typography.Paragraph>
+              Please ensure the following information is correct and up to date. If changes are
+              needed please edit your&nbsp;
+              <Link
+                to={routes.EDIT_PROJECT_SUMMARY.dynamicRoute(
+                  project.project_guid,
+                  project.project_summary.project_summary_guid
+                )}
+              >
+                project description
+              </Link>
+              .
+            </Typography.Paragraph>
+          </Col>
+        </Row>
+        <Row>
+          <Col span={12} style={{ marginRight: "40px" }}>
+            <Typography.Title level={5}>Primary Contact</Typography.Title>
+            <Field
+              id="primary_contact"
+              name="primary_contact"
+              disabled
+              component={renderConfig.FIELD}
+            />
+          </Col>
+          <Col span={11}>
+            <Typography.Title level={5}>Mine Name</Typography.Title>
+            <Field id="mine_name" name="mine_name" disabled component={renderConfig.FIELD} />
+          </Col>
+        </Row>
+        <Typography.Title level={4}>Application Files</Typography.Title>
+        <Typography.Title level={5}>Upload primary application document</Typography.Title>
+        <Typography.Paragraph>
+          Please upload the main document for the submission. If your single document contains all
+          supporting information you may not need to include separate supporting documentation.
+        </Typography.Paragraph>
+        <Field
+          id={MAJOR_MINES_APPLICATION_DOCUMENT_TYPE.PRIMARY}
+          name={MAJOR_MINES_APPLICATION_DOCUMENT_TYPE.PRIMARY}
+          label="Upload primary application document"
+          onFileLoad={(documentName: string, document_manager_guid: string) => {
+            onFileLoad(
+              documentName,
+              document_manager_guid,
+              MAJOR_MINES_APPLICATION_DOCUMENT_TYPE_CODE.PRIMARY,
+              MAJOR_MINES_APPLICATION_DOCUMENT_TYPE.PRIMARY
+            );
+          }}
+          onRemoveFile={(err: any, fileItem: any) => {
+            onRemoveFile(
+              err,
+              fileItem,
+              MAJOR_MINES_APPLICATION_DOCUMENT_TYPE.PRIMARY,
+              primary_documents
+            );
+          }}
+          projectGuid={project.project_guid}
+          labelIdle={
+            '<strong>Drag & Drop your files or <span class="filepond--label-action">Browse</span></strong><div>(Accepted filetypes: .kmx .doc .docx .xlsx .pdf)</div>'
+          }
+          allowMultiple
+          acceptedFileTypesMap={acceptedFileTypesMap}
+          component={MajorMineApplicationFileUpload}
+          uploadType="primary_document"
+          validate={[required]}
```

Comment:
I have a feeling, because it's using the MajorMineApplicationFileUpload, that the validate={[required]} doesn't do anything. Looks like that file uses the newer RenderFileUpload that'll show required and generate the labelIdle for you, but that it only uses it to standardize the uploadUrl and it doesn't pass all the props.